20190519-WebAbstract/BDA_FS19_WebAbstract.tex

```diff
@@ -108,11 +122,24 @@ \section{Ergebnisse}
 	\includegraphics[keepaspectratio,width=2.7cm]{img/HSLU_Logo_Header}
 \end{textblock*}
 
-\section{Lösungskonzept}
+\section{Ergebnisse}
+Wie in der Aufgabenstellung beschrieben wurden zwei Konzepte ausgearbeitet. Das eine konzentrierte sich auf das automatische Auffinden von Exemplaren im Hochregallager selbst, während das andere eine Station voraussieht, welche fehlplatzierte Exemplare vor dem Einlagern identifiert und den betreffenden Behälter ausschleust (in Abbildung \ref{fig:PosAntennen} sind die Antennenposition dieser Station dargestellt).
 
-\section{Spezielle Herausforderungen}
+\begin{figure}[htb]
+	\centering
+	\includegraphics[keepaspectratio,width=\textwidth]{Position_Antennen}
+	\caption{Positionierung der LongRange Antennen des zweiten Konzepts}
+	\label{fig:PosAntennen}
+\end{figure}
 
-\section{Ausblick}
+In einer nächsten Versuchsphase konnte verifiziert werden, dass die Hardware, welche im Moment auf dem Markt existiert für das erste Konzept (der vollautomatischen Suche) nicht geeignet ist, und dieses daher für die Machbarkeitsstudie nicht weiter beachtet wurde.
```

Comment:
In **der darauffolgenden** Versuchsphase konnte verifiziert werden, dass die Hardware, welche im Moment auf dem Markt **existiert, für das Konzept mit der vollautomatischen Suche im Hochregellager, nicht geeignet ist und daher dieses** für die Machbarkeitsstudie nicht weiter beachtet wurde.